kvbc/tools/db_editor/include/kv_blockchain_db_editor.hpp

```diff
@@ -966,13 +983,13 @@ struct ResetMetadata {
       }
     }
 
-    // remove db checkpoint metadata after updating it with empty dbcheckpoint metadata
-    DbCheckpointMetadata dbchkpt_mdt;
-    std::ostringstream outStream;
-    concord::serialize::Serializable::serialize(outStream, dbchkpt_mdt);
-    auto data = outStream.str();
-    std::vector<uint8_t> v(data.begin(), data.end());
-    p->setDbCheckpointMetadata(v);
+    // // remove db checkpoint metadata after updating it with empty dbcheckpoint metadata
+    // DbCheckpointMetadata dbchkpt_mdt;
+    // std::ostringstream outStream;
+    // concord::serialize::Serializable::serialize(outStream, dbchkpt_mdt);
+    // auto data = outStream.str();
+    // std::vector<uint8_t> v(data.begin(), data.end());
```

Comment:
?

Reply:
need to be removed, we do want to save the recovered knowledge about the old db checkpoints